### PR TITLE
Command Query Separation

### PR DESCRIPTION
**Pull request by Juules32**
- this is done to separate between commands and queries

### CODE REVIEW

File: src/Chirp.Razor/Shared/Storage/ChirpStorage.cs [hunk 1]
```diff
@@ -40,29 +41,16 @@ public void StoreCheeps(List<Cheep> entities)
...
-            .Where(c => c.Author.Name == author)
-            .Include(c => c.Author)
-            .Take(amount)
-            .ToList();
-
-        return cheeps;
-    }
-
-    public IEnumerable<Cheep> GetCheepsPerPage(int pageNumber, int amount)
+    public List<Cheep> QueryCheeps(int pageNumber, int amount, string? author = null)
```
Comment (AlbertRossJoh) [inline src/Chirp.Razor/Shared/Storage/ChirpStorage.cs:44]: Please change to IEnumerable.
This is done here and in the interface. You also need a tolist in cheepservice.

**Review by AlbertRossJoh — CHANGES_REQUESTED** [review]
Very cool, one comment

**Review by sumrdev — APPROVED** [review]
crazy quick fix lets get this merged :100:

**Review by AlbertRossJoh — APPROVED** [review]
LGTM

**Pull request merged**